docs/install/build_from_source.md

```diff
@@ -16,6 +16,8 @@ This document explains how to build MXNet from source code. Building MXNet from
 
 You need C++ build tools and a BLAS library to build the MXNet shared library. If you want to run MXNet with GPUs, you will need to install [NVDIA CUDA and cuDNN](https://developer.nvidia.com/cuda-downloads) first.
 
+You may use [GNU Make](https://www.gnu.org/software/make/) to build the library but [cmake](https://cmake.org/) is required
```

Comment:
when building with make with the MKLDNN, cmake it required: https://github.com/apache/incubator-mxnet/blob/master/mkldnn.mk#L40. 

additionally there two discussions already to remove the makefile and make cmake default 

https://lists.apache.org/list.html?dev@mxnet.apache.org:2018-6:cmake
https://github.com/apache/incubator-mxnet/issues/8702

Reply:
the purpose of the vote was to eliminate the Makefile. if you think we should keep the Makefile, raise it in the email list. otherwise the direction we are heading is removing it completely and becoming dependent on cmake.